portfolio/src/main/java/com/google/sps/servlets/ListCommentsServlet.java

```diff
@@ -57,4 +60,28 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
     response.setContentType("application/json;");
     response.getWriter().println(gson.toJson(comments));
   }
+
+  /**
+   * Convert a string into a integer. If not a natural number,
+   * return 0 with an error message.
+   * @param stringNum A string that will be parsed into a natural number
+   * @return The number as an int, 0 if input is invalid
+   */
+  private static int parseNaturalNumber(String stringNum) {
+    int num = 0;
+
+    try {
+      num = Integer.parseInt(stringNum);
+    } catch (NumberFormatException e) {
+      System.err.println("Did not input a valid integer in String form.");
```

Comment:
nit: How about combining the two error log statements into a single statement?

`System.err.println("Did not input a valid integer in String form." + e);`

Reply:
Fixed!